README.md

```diff
@@ -1094,6 +1094,18 @@ Here `pick_data` is the data to simplify and currently only supports `MultiSyste
 
 
 ## Set up machine
```

Comment:
As far as i understand when set `api_version < 1.0`, the old-style `machine.json` is parsed, while for `api_version >= 1.0`, the user should write the `machine.json` compatible to the `dpdispatcher` package. 

1, why not using a key like `compatible_to_old_dispatcher` to indicate which dispatcher should be used to parse the `machine.json`
2, the current documentation is very unclear, nobody can understand what he/she should do with `machine.json` when swithchiing to new version of dpgen.